app/pages/new-dose-medication/new-dose-medication.html

```diff
@@ -3,72 +3,58 @@
 </ion-header>
 
 <ion-content padding  class="new-dose-medication">
+
     <form (ngSubmit)="onSubmit()"
           #requestForm="ngForm">
+
         <ion-list>
+
             <ion-item>
-                <ion-label floating>Medication Name</ion-label>
+                <ion-label floating>
+                    Medication Name
+                </ion-label>
                 <ion-input type="text"
                            [(ngModel)]="doseSeries.med.name"
-                           required></ion-input>
+                           required>
+                </ion-input>
             </ion-item>
 
 
             <ion-item>
-                <ion-label floating>Dose Amount</ion-label>
+                <ion-label floating>
+                    Dose Amount
+                </ion-label>
                 <ion-input type="number"
                            [(ngModel)]="doseSeries.med.doseAmount"
-                           required></ion-input>
+                           required>
+                </ion-input>
             </ion-item>
 
 
             <ion-item>
-                <ion-label floating>Dose Units (pills, mg, cc, etc.)</ion-label>
+                <ion-label floating>
+                    Dose Units (pills, mg, cc, etc.)
+                </ion-label>
                 <ion-input type="text"
-                           [(ngModel)]="doseSeries.med.doseUnit"></ion-input>
-            </ion-item>
-
-            <!--ion-item>
-                <ion-label>Sunday</ion-label>
-                <ion-checkbox checked="false"></ion-checkbox>
+                           [(ngModel)]="doseSeries.med.doseUnit">
+                </ion-input>
             </ion-item>
 
             <ion-item>
-                <ion-label>Monday</ion-label>
-                <ion-checkbox checked="false"></ion-checkbox>
+                <ion-label floating>
+                    Days of Week
+                </ion-label>
+                <ion-select [(ngModel)]="doseSeries.daysOfWeek"
+                            multiple="true"
+                            cancelText="Cancel"
+                            okText="Select">
+                    <ion-option *ngFor="let dayOption of daysOfWeekOptions"
+                                [value]="dayOption.value">
+                        {{dayOption.display}}
+                    </ion-option>
+                </ion-select>
```

Comment:
This is the select dialog for picking the days.  It starts with every day set as the default.